cranelift/codegen/src/isa/x64/lower.isle

```diff
@@ -1443,6 +1443,21 @@
 (rule (lower (resumable_trap code))
       (side_effect (x64_ud2 code)))
 
+;;;; Rules for `return` and `fallthrough_return` ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(rule (lower (return args))
```

Comment:
Could we move the comment here "N.B.: the Ret itself is generated by the ABI" from the original? Otherwise it's somewhat unclear to the reader why `return` and `fallthrougth_return` should be identical, and why they only set up the return value rather than really returning.

Reply:
Great point, I'll add that.